src/fs/driver/initfs/initfs_dvfs.c

```diff
@@ -246,5 +246,5 @@ static struct dumb_fs_driver initfs_dumb_driver = {
 	.mount_end = initfs_mount_end,
 };
```

Comment:
And what's about making declarations of the rest `dumb_drv_tab` elements `const` too?